script/munin-httpd

```diff
@@ -61,7 +58,5 @@ sub handle_request
 
 package main;
 
-$ENV{PATH} = '/usr/bin:/bin';
```

Comment:
Same here: I am not sure, why the PATH was limited up to now. Probably preventing unwanted locations of programs to be executed directly?
I have the feeling, that the current restriction is not useful (i.e. would be a surprise for installations with non-packaged installations of rrdtool or similar). But I am not sure about its purpose.

Reply:
Exactly.

Reply:
Do you mean, we should remove the PATH override?
(I would think so)

Reply:
Yes.